src/test/java/com/scottlogic/AuthorPostSorterTest.java

```diff
@@ -0,0 +1,76 @@
+package com.scottlogic;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class AuthorPostSorterTest {
+    @Test
+    void nullTest() {
+        List<UserPost> actual = new AuthorPostSorter().sort(null);
+
+        assertNull(actual);
+    }
+
+    @Test
+    void emptyListTest() {
+        List<UserPost> actual = Collections.emptyList();
+        List<UserPost> expected = Collections.emptyList();
+
+        actual = new AuthorPostSorter().sort(actual);
+
+        assertEquals(expected, actual);
+    }
+
```

Comment:
There are a few easier test cases that you might want to do before moving onto 3 length lists